chapters/en/slides/module2/module2_01.md

```diff
@@ -0,0 +1,184 @@
+talk](https://www.youtube.com/watch?v=hsfWtPH2kDg), who is visualization
+researcher at the university of Washington and whose research group
+created D3 and VegaLite (which Altair is based on).
+
+---
+
+## Summary of visual channel efficiency
+
+<img src="/module2/visual-channels-rank.png" style="width:80.0%" />
+
+Notes: Even if you got both these right yourself, the fact that many
+people prefer one over the other means that in order for you to create
+effective visualizations you need to know which visual channels are the
+easiest for humans to decode. Luckily, there has been plenty of research
+in this area, which can be summarizes in the schematic on this slide.
+
+Position is by far the best and therefore we should put our most
+important comparison there. Using position often means that we can’t use
+other things such as length or angle (like the angle in a pie chart),
+but we can add size or color to represent other relationships. Even if
+it is hard to tell exact information from these (is this color/dot 2x
+darker/bigger than another?) they are good to give and idea of trends in
+the data (are these dots darker/bigger than the others?
+
+---
+
+## Don’t use 3D without a good reason
+
+### Unnecessary 3D makes plot interpretation harder
+
+![](/module2/bad-3d-barchart.png)
+
+Notes: The biggest issues with using 3D is when it is used unnecessarily
+(like a 3D bar or pie chart), as the only way to compare position (like
+a 3D scatter plot), and when they are represented on a 2D medium like a
+paper where they can’t be rotated. In this slide we see a 3D bar chart,
+where it looks like the values of the bars are around A=0.7, B=1.7,
+C=2.7 and D=3.7. However, this is only because of the angle of the
+camera in the plot, the actual values here of A, B, C, and D are
+actually 1, 2, 3, and 4, respectively.
+
+---
+
+## Meaningful 3D can facilitate plot interpretation
+
+![](https://media.github.ubc.ca/user/1751/files/da4e1b00-0fd3-11eb-940d-0b06a28a0186)
+
+Notes: Sometimes 3D can be useful, like a topographical map or a protein
+folding visualization. But be cautious, we saw above that even in
+naturally 3D systems like blood vessels it is still mentally more
+complex when they are 3D. [Claus Wilke’s has a good
+chapter](https://clauswilke.com/dataviz/no-3d.html) on this if you are
+interested to read more.
+
+There is also some [interesting work done with the Rayshader
+library](https://www.tylermw.com/3d-ggplots-with-rayshader/) that maps
+3D in an intuitive way and incorporates reasonable camera rotation
+around the objects. This slide contains is an example of visualizing the
+bend in space time via the 3D depth channel, eliminating the need for an
+additional 2D plot.
+
+---
+
+## Properly designed visualizations help saving lives
+
+![](/module2/blood-vessels-40.png)
+
+Notes: How much these best practices actually matters might be a bit
+abstract until you gain personal experience from it, Therefore, I want
```

Comment:
Again, not sure personal experience is the right term here either (neither is practice...)